generators/app/templates/src/app/components/ErrorBoundary/i18n.ts

```diff
@@ -0,0 +1,11 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'ERRORBOUNDARY', {
+  TITLE: '¡Oh no, a ocurrido un error!',
+  TRY_AGAIN: 'Intenta nuevamente',
+  BACK_TO_HOME: 'Volver al inicio',
+  ERROR_ALERT: 'Unexpected error occurred',
```

Comment:
before you are talking in spanish, and here in english. Only use spanish translations on `es` translations